MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+# MAINTAINERS
+
+## Current Maintainers
+
+| GitHub Username | Name |
+| --- | --- |
+| **fibonacci1729** | Brian Hardock |
```

Comment:
@michelleN do we want to do the same note about alphabetical order here?

Reply:
updated! thanks!